tgui/packages/tgui/interfaces/NanoCrucible.tsx

```diff
@@ -0,0 +1,95 @@
+/**
+ * @file
+ * @copyright 2025
+ * @author FlameArrow57 (https://github.com/FlameArrow57)
+ * @license ISC
+ */
+
+import { Button, Flex, Icon, Section } from 'tgui-core/components';
+
+import { useBackend } from '../backend';
+import { Window } from '../layouts';
+
+interface PacketInfo {
+  first_part;
+  first_part_img;
+  second_part;
+  second_part_img;
+  result_part;
```

Comment:
Again, these should have type hints even if they're all just strings

Reply:
Sure, are images passed considered strings?

Reply:
yes, they're base64 strings